libc-test/build.rs

```diff
@@ -1809,6 +1810,22 @@ fn test_freebsd(target: &str) {
             // base system anyway.
             "CTL_MAXID" | "KERN_MAXID" | "HW_MAXID" | "USER_MAXID" => true,
 
+            // This was renamed in FreeBSD 12 and 13 (r352486).
+            "CTL_UNSPEC" | "CTL_SYSCTL" => true,
+
+            // These were added in FreeBSD 12 and 13 (r352486),
+            // but they are just names for magic numbers that existed for ages.
+            "CTL_SYSCTL_DEBUG"
+            | "CTL_SYSCTL_NAME"
+            | "CTL_SYSCTL_NEXT"
+            | "CTL_SYSCTL_NAME2OID"
+            | "CTL_SYSCTL_OIDFMT"
+            | "CTL_SYSCTL_OIDDESCR"
+            | "CTL_SYSCTL_OIDLABEL" => true,
+
+            // This was renamed in FreeBSD 12 and 13 (r350749).
+            "IPPROTO_SEP" | "IPPROTO_DCCP" => true,
```

Comment:
I think we could use `freebsd_ver`.

Reply:
Not really: this would change between minor versions (12.1 and the upcoming 12.2)

Reply:
Ah, I see. Then it'd be great if the comment mentions that.